message/ops.go

```diff
@@ -22,8 +22,8 @@ const (
 	PingOp Op = iota
 	PongOp
 	HandshakeOp
+	GetPeerListOp
```

Comment:
Confirming it is ok to re-arrange this order

Reply:
The ops are not sent over the wire (anymore)